crates/astria-conductor/src/executor/tests.rs

```diff
@@ -165,41 +180,69 @@ async fn start_mock() -> MockEnvironment {
     }
 }
 
+async fn start_mock_with_optimism_handler() -> (
```

Comment:
It's easier to read if you turn the tuple into a a struct where you give these names.